src/util/common-translation.ts

```diff
@@ -6,7 +6,8 @@ import { translationMetadata } from "../resources/translations-metadata";
 import { HomeAssistant } from "../types";
 
 const DEFAULT_BASE_URL = "/static/translations";
-const STORAGE = window.localStorage || {};
+const STORAGE =
+  (typeof window !== "undefined" && window.localStorage) || ({} as Storage);
```

Comment:
Window is not available in a test env

Reply:
Add `global.window` to <https://github.com/home-assistant/frontend/blob/dev/test/setup.js>?

Reply:
Yes, that's better ! 👍